arches/app/models/graph.py

```diff
@@ -568,14 +568,15 @@ def delete_instances(self, verbose=False):
         deletes all associated resource instances
 
         """
+        resources_to_delete = list(Resource.objects.filter(graph_id=self.graphid))
+        delete_count = len(resources_to_delete)
         if verbose is True:
-            bar = pyprind.ProgBar(Resource.objects.filter(graph_id=self.graphid).count())
-        for resource in Resource.objects.filter(graph_id=self.graphid):
+            bar = pyprind.ProgBar(delete_count)
+        for resource in resources_to_delete:
             resource.delete()
             if verbose is True:
                 bar.update()
-        if verbose is True:
-            print(bar)
```

Comment:
Sorry, do you mind sharing why this was removed?

Reply:
I'm showing that update() only prints progress, not start/finish/elapsed like print() does.

Reply:
sorry that was my mistake, I thought it was just an older usage of progbar. I've reverted this commit and pushed to the branch.